avocado/core/nrunner.py

```diff
@@ -18,6 +18,8 @@
 import unittest
 from uuid import uuid1
 
+from .runners.utils import messages
```

Comment:
It's unfortunate that we still don't have a functional test for the `podman` spawner (or a CI check), but this breaks the current implementation.  We are researching the idea of using eggs to deploy Avocado to the container, but this has not been validated/implemented yet.

So, while this is my main concern because it breaks the `podman` spawner, it also makes me believe that these methods should be available, from the start, to all Python based runner implementations.  That is, I see no reason, besides maybe the because we'd have a larger file, that most of the `messages` functions can't be static methods of the `BaseRunner` class.

Reply:
Yes, my mistake, I wasn't aware that we don't have functional tests for podman spawner, so I didn't test it. I will investigate it further and find a solution.